KarhooUISDK/Screens/BookingScreen/KarhooBookingPresenter.swift

```diff
@@ -31,8 +31,8 @@ final class KarhooBookingPresenter {
     private let vehicleRulesProvider: VehicleRulesProvider
     private let router: BookingRouter
 
-    private var coverageCache: [String: Bool] = [:]
-    private var coverageCheckInProgress: [String] = []
+//    private var coverageCache: [String: Bool] = [:]
```

Comment:
Leftover?

Reply:
Yes. I left this to be deleted later when we move things around, but I can also remove them now if you want. Nothing broke with these two lines out of the way.